coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,19 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = '.hs',
+    versions = 1.0, 1.1, 1.2, 1.3, 1.4, 98, 2010
+    special_chars = list('!#$%&*+./<=>?@\\^|-~')
+    comment_delimiters = '--',
+    multiline_comment_delimiters = {'{-': '-}'}
+    keywords = [
+        '-<', '-<<', '::', ';', '<-', ',', '_', '{, }',
```

Comment:
`'{, }'` should be `'{',  '}'` ...?

Reply:
I think it is right because opening and closing bracket is one keyword.If we use opening bracket then it is compulsory to use close bracket. They can not used separately.

Reply:
Your point makes sense, but this: 

`{, }`

is definitely not a keyword. Maybe wait for other reviewers' voice.

Reply:
There is no such `{, }` keyword in Haskell. It should be `'{', '}'` because it is the same as [multiline_comment_delimiters](https://wiki.haskell.org/Keywords#.7B-.2C_-.7D). You write it `'{-', '-}'` instead of `'{-, -}'`, right?

Reply:
I will removed this because it came in  multiline_comment_delimiters.

Reply:
> I will removed this because it came in multiline_comment_delimiters.

Actually that is not what I means. However, it is fine.